### ISSUE TITLE
Version 5.3.0 throw an error on github action

### ISSUE THREAD

**Issue by Trosss** [HubSpot/hubspot-cli#1134]
## Description and Context
When running `npx --package=@huspot/cli@latest` on a github action it throws an error:
`/usr/bin/env: ‘node --no-deprecation’: No such file or directory`

### Steps to reproduce
Create a github action and add the following step:
```
 - name: Deploy
   env:
      HUBSPOT_PORTAL_ID: <hubspot_portal_id>
      HUBSPOT_PERSONAL_ACCESS_KEY: <hubspot_personal_access_key>
   run: |
      npx --yes --package=@hubspot/cli@latest --call='hs upload <source> <destination> --use-env'
   shell: bash
```

## Expected behavior
It shouldn't throw an error

**Comment by cosmingeorge8** [comment]
I am facing the same issue, running HubSpot/hubspot-cms-deploy-action@v1.7.

**Comment by brandenrodgers** [comment]
Thanks you all for surfacing this. We just releases a patch fix for this (5.3.1) that removes the `--no-deprecation` flag. I'm going to close this issue, but feel free to re-open it if you're still having issues.
